tests/ci/codebuild/release/stage.yml

```diff
@@ -21,17 +21,16 @@ phases:
       # Debugging only
       - aws sts get-caller-identity
       # Retrieve encrypted configuration from S3
-      # - aws s3 cp s3://${CONFIG_BUCKET}/${JKS_FILENAME}.enc $HOME/${JKS_FILENAME}.enc
-      # - aws s3 cp s3://${CONFIG_BUCKET}/${GPG_FILENAME} $HOME/${GPG_FILENAME}.enc
-      # - aws s3 cp s3://${CONFIG_BUCKET}/${PROPERTIES_FILENAME}.enc $HOME/${PROPERTIES_FILENAME}.enc
+      - aws s3 cp s3://${CONFIG_BUCKET}/${JKS_FILENAME}.enc $HOME/${JKS_FILENAME}.enc
+      - aws s3 cp s3://${CONFIG_BUCKET}/${GPG_FILENAME} $HOME/${GPG_FILENAME}.enc
+      - aws s3 cp s3://${CONFIG_BUCKET}/${PROPERTIES_FILENAME}.enc $HOME/${PROPERTIES_FILENAME}.enc
       # Decrypt configuration
       - mkdir -p $HOME/.gradle
-      # - aws-encryption-cli --decrypt -S --wrapping-keys key=${KMS_ARN} --commitment-policy require-encrypt-allow-decrypt -i $HOME/${JKS_FILENAME}.enc -o $HOME/.gradle/${JKS_FILENAME} --encryption-context filename=${JKS_FILENAME}
-      # - aws-encryption-cli --decrypt -S --wrapping-keys key=${KMS_ARN} --commitment-policy require-encrypt-allow-decrypt -i $HOME/${GPG_FILENAME}.enc -o $HOME/${GPG_FILENAME} --encryption-context filename=${GPG_FILENAME}
-      # - aws-encryption-cli --decrypt -S --wrapping-keys key=${KMS_ARN} --commitment-policy require-encrypt-allow-decrypt -i $HOME/${PROPERTIES_FILENAME} -o $HOME/.gradle/gradle.properties --encryption-context filename=${PROPERTIES_FILENAME}
-      # - tar -xzvpf $HOME/${GPG_FILENAME}
-      # TODO: Add actual publishing logic
-      - ./gradlew -DFIPS=${FIPS} fakePublish
+      - aws-encryption-cli --decrypt -S --wrapping-keys key=${z} --commitment-policy require-encrypt-allow-decrypt -i $HOME/${JKS_FILENAME}.enc -o $HOME/.gradle/${JKS_FILENAME} --encryption-context filename=${JKS_FILENAME}
```

Comment:
does somebody set `${z}`? seems like an odd env variable name....

Reply:
Copy-paste error. I'll merge since this doesn't pose a risk to breaking real things and then fix it myself.